services/QuillLMS/client/app/bundles/Teacher/components/scorebook/premium_banners/free_trial_status.jsx

```diff
@@ -14,6 +14,15 @@ export default class extends React.Component {
   };
 
   render() {
+    const premiumButton = this.props.originPage == 'premium' ?
```

Comment:
We should destructure `this.props` calls. There's a staging linter [rule](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/destructuring-assignment.md) to that effect.